java-samples/corda5-negotiation-cordapp/settings.gradle

```diff
@@ -0,0 +1,36 @@
+pluginManagement {
+    // Declare the repositories where plugins are stored.
+    repositories {
+        gradlePluginPortal()
+        mavenCentral()
+        mavenLocal()
+        maven {
+            url = "$artifactoryContextUrl/corda-os-maven"
+            authentication {
+                basic(BasicAuthentication)
+            }
+            credentials {
+                username = settings.ext.find('cordaArtifactoryUsername') ?: System.getenv('CORDA_ARTIFACTORY_USERNAME')
+                password = settings.ext.find('cordaArtifactoryPassword') ?: System.getenv('CORDA_ARTIFACTORY_PASSWORD')
+            }
+        }
+    }
```

Comment:
any reference to artifactory is strictly internal now. you don't need this.

Reply:
Sometimes, cordapps pulled from CSDE use things from development branch, which is not necessarily useable by external cordapp developers because they use artifacts not released publicly